homeassistant/components/homematic.py

```diff
@@ -0,0 +1,858 @@
+"""
```

Comment:
Is this components/homematic/__init__.py? If so, why did you copy it here?

Reply:
I checked out current master and made my changes. In current master, this file still exists. May be dropped if not needed.